homeassistant/components/eq3btsmart/climate.py

```diff
@@ -111,7 +116,16 @@ def precision(self):
 
     @property
     def current_temperature(self):
-        """Can not report temperature, so return target_temperature."""
+        """Can not report temperature, so return source temperature.
+        Return target temperature if no source present.
+        """
+        if self._source:
+            state = self.hass.states.get(self._source)
```

Comment:
We don't allow device interfacing integrations to consume states from other integrations. That is reserved for integrations that only consumes other states and doesn't interface with devices directly, like template integrations.

Reply:
So would this be something that could be integrated into the climate frontend integration instead?

Reply:
No, it should set up the sensor platform and provide its own based on the same data.